src/EditorFeatures/Core.Wpf/InlineRename/Taggers/ClassificationTypeDefinitions.cs

```diff
@@ -16,17 +16,14 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
 {
     internal sealed class ClassificationTypeDefinitions
     {
-        // Only used for theming, does not need localized
-        public const string InlineRenameField = "Inline Rename Field Text";
-
         [Export]
-        [Name(InlineRenameField)]
+        [Name(RenameClassificationTypeNames.InlineRenameField)]
         [BaseDefinition(PredefinedClassificationTypeNames.FormalLanguage)]
         internal ClassificationTypeDefinition InlineRenameFieldTypeDefinition;
```

Comment:
This should move down to EditorFeatures as well, as the definition of the classification type can be done without WPF. Otherwise I worry the call to GetClassificationType may return null on the Cocoa side.

Reply:
There is a duplicate definition for this in EditorFeatures.Cocoa, but I can move down to consolidate.